Added Function to undraw all entities

Feuerbowle opened the pull request:
New function to undraw all entities.

I only have one problem with the test routine. I've found no way to check if an entitie is visible or not. Maybe anyone knows a way to check this.

I also want to add the functionality to undraw entitie groups like all NCPrograms or all Toolpaths as well as single entities themself. At the moment a see three different approaches:

Version 1:
powermill.UndrawAllNCPrograms();			//Undraw all ncprograms

Version 2:
powermill.Undraw(NCProgram);				//Undraw all ncprograms
powermill.Undraw(NCProgram, [PMNCProgram]);	//Undraw NC program [PMNCProgram]

Version 3:
session.NCPrograms.UndrawAll()				//Undraw all ncprograms
session.Boundarys.Undraw([PMNCProgram])		//Undraw NC Program [PMNCProgram]

With Version 2 and Version 3 i would like to introduce a new enum like "EntitieTypes" so you can name the entities you would like to undraw.

Feuerbowle committed:
Added Function to undraw all entities

New function to undraw all entities.

I only have one problem with the test routine. I've found no way to check if an entitie is visible or not. Maybe anyone knows a way to check this.

I also want to add the functionality to undraw entitie groups like all NCPrograms or all Toolpaths as well as single entities themself. At the moment a see three different approaches:

Version 1:
powermill.UndrawAllNCPrograms();			//Undraw all ncprograms

Version 2:
powermill.Undraw(NCProgram);				//Undraw all ncprograms
powermill.Undraw(NCProgram, [PMNCProgram]);	//Undraw NC program [PMNCProgram]

Version 3:
session.NCPrograms.UndrawAll()				//Undraw all ncprograms
session.Boundarys.Undraw([PMNCProgram])		//Undraw NC Program [PMNCProgram]

With Version 2 and Version 3 i would like to introduce a new enum like "EntitieTypes" so you can name the entities you would like to undraw.

lukeedw commented on Delcam.ProductInterface.PowerMILL.Test/PMAutomationTest.cs:
```diff
@@ -247,5 +247,15 @@ public void GetListOfPmComObjectsTest()
+            var testPattern = _powerMill.ActiveProject.Patterns.CreatePattern(Files.TestFiles.CurvesFiles);
+            testPattern.IsVisible = true;
+            _powerMill.UndrawAll();
+            //Assert.False(testPattern.IsVisible);                     
```
I don't know either...  If you want to do "Assert.Pass()" then I would be ok with that!

Feuerbowle committed:
Update PMAutomationTest.cs

"Assert.Pass();" always throws an exeption, so i used "Assert.IsTrue(true);" instead. Is this also Okay?

What about the other "undraw" functions? What way should i go?
